models/dex/dex_prices_latest_legacy.sql

```diff
@@ -1,7 +1,7 @@
 {{ config(
-        alias = alias('prices_latest'),
+        alias = alias('prices_latest', legacy_model=True),
```

Comment:
this is needed for when we add to dunesql to help differentiate table names

Reply:
ah got it, thanks Jeff!